src/state_transition/AdmistrationSetters.sol

```diff
@@ -44,22 +74,26 @@ contract AdmistrationSetters is LTVState, IAdministrationErrors, IAdministration
         emit MaxTotalAssetsInUnderlyingChanged(oldValue, _maxTotalAssetsInUnderlying);
     }
 
-    function _setMaxDeleverageFee(uint256 value) internal {
-        require(value <= 10 ** 18, InvalidMaxDeleverageFee(value));
-        uint256 oldValue = maxDeleverageFee;
-        maxDeleverageFee = value;
-        emit MaxDeleverageFeeChanged(oldValue, value);
+    function _setMaxDeleverageFee(uint16 dividend, uint16 divider) internal {
+        require(dividend >= 0 && dividend <= divider, InvalidMaxDeleverageFee(dividend, divider));
+        uint16 oldDividend = maxDeleverageFeeDividend;
+        uint16 oldDivider = maxDeleverageFeeDivider;
+        maxDeleverageFeeDividend = dividend;
+        maxDeleverageFeeDivider = divider;
+        emit MaxDeleverageFeeChanged(oldDividend, oldDivider, dividend, divider);
     }
 
     function _setIsWhitelistActivated(bool activate) internal {
         require(!activate || address(whitelistRegistry) != address(0), WhitelistRegistryNotSet());
-        bool oldValue = isWhitelistActivated;
-        isWhitelistActivated = activate;
+        bool oldValue = isWhitelistActivated();
+        boolSlot = activate
```

Comment:
Mb it's good idea to create state transition file where you put all this things in BoolUpdate file where bool magic will be abstracted.